<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>0</y>
     <width>211</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Server</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="listenButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>开始监听</string>
   </property>
  </widget>
  <widget class="QLabel" name="listenStatus">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>230</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>online</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>220</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>offline</string>
   </property>
  </widget>
  <widget class="QListWidget" name="onlineList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>261</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="offlineList">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>270</y>
     <width>261</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <zorder>listenButton</zorder>
  <zorder>label</zorder>
  <zorder>listenStatus</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>onlineList</zorder>
  <zorder>offlineList</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
